policy/apps.py

```diff
@@ -21,9 +23,18 @@
     "ACTIVATION_OPTION_CONTRIBUTION": 1,
     "CTIVATION_OPTION_PAYMENT": 2,
     "ACTIVATION_OPTION_READY": 3,
-    "contribution_receipt_length": 5
+    "contribution_receipt_length": 5,
+    "control_family_level": False
```

Comment:
The naming is not explicit enough. If it is about enabling the polygamic family then you should name this parameter like `enable_polygamic_family` to remove ambiguity. 
Make sure you list this configuration parameter in readme.md.

Reply:
I just fixed it now